### ISSUE TITLE
Option to ignore newlines

### ISSUE THREAD

**Issue by bendavis78** [vslavik/poedit#665]
Poedit is showing the following warning:

![image](https://user-images.githubusercontent.com/324392/98742007-49205900-2373-11eb-8002-ead42d206fe3.png)

Is there a way to make it ignore newlines for warnings like this?

If I try to add a newline on the translation, it gets trimmed off when I click "Save", and the error persists.

**Comment by vslavik** [comment]
> Is there a way to make it ignore newlines for warnings like this?

No. That's the *point* of the warning, to warn about differences in whitespace. The thing you're translating may be relying on the trailing newline and not having it there could break things.

> If I try to add a newline on the translation, it gets trimmed off when I click "Save", and the error persists.

*This* is bad, though, and would say is the actual underlying issue. Could you share your file as a real-life example? 

Thanks!

**Comment by vslavik** [comment]
Your screenshot shows that there's some significant whitespace both before "Click" and on the last line. Either it's not a space (with which what you complain about is not reproducible) or you do something different from what is the reasonable interpretation (which is that you type newline and the same whitespace as in the source text). 

Can you please clarify reproduction steps, including a file to use, that one has to follow to see this issue? Issues are two-way communication, not write-only dumps, so I'd appreciate if you could provide some cooperation to the people who pay attention to the issue you filed. TIA!
